<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingForm</class>
 <widget class="QWidget" name="TrainingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>224</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="level1Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>200</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="level2Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>200</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="level3Button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>200</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="level4Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>200</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="level5Button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>200</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border: none; padding: 0; margin: 0; background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="niv1Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="niv2Label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="niv3Label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="niv4Label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="niv5Label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau 5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
